pkg/client/group_mgmt_client.go

```diff
@@ -157,7 +162,12 @@ func (client *GroupMgmtClient) Put(path, id string, payload interface{}, respHol
 		// http code 202, handle async job
 		if response.StatusCode() == 202 {
 			// extract error message
-			return processAsyncResponse(client, response.Body())
+			id, err := processAsyncResponse(client, response.Body())
+			if id != nil {
+				return client.Get(path, id.(string), respHolder)
+			} else {
+				return nil, err
+			}
```

Comment:
Common code again.  Can this be moved to processAsyncResponse?

Reply:
Not much of a common code otherwise we will have to pass "path", respHolder additional argument to processAsyncResponse().